jobbergate-api/pyproject.toml

```diff
@@ -72,6 +71,9 @@ psycopg2 = "^2.9.5"
 [tool.poetry.scripts]
 dev-tools = "dev_tools:app"
 
+[tool.poetry.group.dev.dependencies]
```

Comment:
Another group for dev dependecies? I see `[tool.poetry.dev-dependencies]` above.

Reply:
Oh, I didn't notice that happened! I will correct it.

Reply:
Addressed in commit ff3f979